astropy/table/tests/test_column.py

```diff
@@ -783,7 +783,7 @@ def test_unicode_sandwich_compare(class1, class2):
     """
     obj1 = class1([b'a', b'c'])
     if class2 is str:
-        obj2 = 'a'
+        obj2 = str('a')
```

Comment:
@mhvk - This is an ugly workaround for the unicode literal testing. Frankly, I don't fully understand why a string caused an issue here, but not when it was in a list below.